package.json

```diff
@@ -41,14 +42,14 @@
     "eslint": "^4.17.0",
     "filesize": "^3.6.1",
     "flow-bin": "0.71.0",
-    "gzip-size": "^4.1.0",
```

Comment:
Moved this to `packages/gestalt/package.json` since its only used in that one workspace